src/main/java/org/spongepowered/common/mixin/core/entity/EntityLivingBaseMixin.java

```diff
@@ -1169,5 +1172,16 @@ private void onResetActiveHand(final CallbackInfo ci) {
     }
 
     // End implementation of UseItemStackEvent
+    
+    @ModifyVariable(method = "heal", index = 1, at = @At("HEAD"))
+    private float sponge$fireHealEntityEvent(float original) {
```

Comment:
I'd rather not start stubbing bare implementation events when we can properly have a real event that will actually work. None of the methods, objects, or types are being supported by simply throwing an event here, so it stands to confusion why the API says one thing, and the implementation is flat out implementing it "blindly" without the remaining objects to support the event.